diff --git a/dist/_browser-chunks/chunk-SZQXB3JV.js b/dist/_browser-chunks/chunk-SZQXB3JV.js
index 3cfc32a9d03ab3a77dc18035cd9fe7e3de03de52..827d00a7a46d09a93276d34cdd85e5fef402bc5d 100644
--- a/dist/_browser-chunks/chunk-SZQXB3JV.js
+++ b/dist/_browser-chunks/chunk-SZQXB3JV.js
@@ -1771,7 +1771,7 @@
       }, onUnhandledRejection = (event) => {
         event.reason && unhandledErrors.add(event.reason);
       };
-      if (this.renderOptions.autoplay && forceRemount && playFunction && this.phase !== "errored") {
+      if (this.renderOptions.autoplay && (initial ? this.story?.parameters?.autoplay ?? false : true) && forceRemount && playFunction && this.phase !== "errored") {
         window?.addEventListener?.("error", onError), window?.addEventListener?.("unhandledrejection", onUnhandledRejection), this.disableKeyListeners = !0;
         try {
           if (isMountDestructured ? await playFunction(context) : (context.mount = async () => {
diff --git a/dist/preview/runtime.js b/dist/preview/runtime.js
index 8f604257f47817b904d6005bf20d10a1ade6de00..1d69e9b54efe7709d5a9b9bd61b5727387a53b09 100644
--- a/dist/preview/runtime.js
+++ b/dist/preview/runtime.js
@@ -34023,7 +34023,7 @@
       }, onUnhandledRejection = (event) => {
         event.reason && unhandledErrors.add(event.reason);
       };
-      if (this.renderOptions.autoplay && forceRemount && playFunction && this.phase !== "errored") {
+      if (this.renderOptions.autoplay && (initial ? this.story?.parameters?.autoplay ?? false : true) && forceRemount && playFunction && this.phase !== "errored") {
         window?.addEventListener?.("error", onError), window?.addEventListener?.("unhandledrejection", onUnhandledRejection), this.disableKeyListeners = !0;
         try {
           if (isMountDestructured ? await playFunction(context) : (context.mount = async () => {
