diff --git a/dist/_browser-chunks/chunk-M6YZR3ZW.js b/dist/_browser-chunks/chunk-M6YZR3ZW.js
index 9cc8051ce3da4a375b0b4fce80179987b6cf14d8..38eab65179f4370a3d369084d9f8315aeb9ad197 100644
--- a/dist/_browser-chunks/chunk-M6YZR3ZW.js
+++ b/dist/_browser-chunks/chunk-M6YZR3ZW.js
@@ -1665,7 +1665,7 @@ var StoryRender = class {
       }, onUnhandledRejection = (event) => {
         event.reason && unhandledErrors.add(event.reason);
       };
-      if (this.renderOptions.autoplay && forceRemount && playFunction && this.phase !== "errored") {
+      if (this.renderOptions.autoplay && (initial ? this.story?.parameters?.autoplay ?? false : true) && forceRemount && playFunction && this.phase !== "errored") {
         window?.addEventListener?.("error", onError), window?.addEventListener?.("unhandledrejection", onUnhandledRejection), this.disableKeyListeners = !0;
         try {
           if (isMountDestructured ? await playFunction(context) : (context.mount = async () => {
diff --git a/dist/preview/runtime.js b/dist/preview/runtime.js
index 3cad9618aae48405ca285689e59b33ae9838cb58..871c689b3ec5526fec74ae19127678f9baf145a2 100644
--- a/dist/preview/runtime.js
+++ b/dist/preview/runtime.js
@@ -35645,7 +35645,7 @@ var StoryRender = class {
       }, onUnhandledRejection = (event) => {
         event.reason && unhandledErrors.add(event.reason);
       };
-      if (this.renderOptions.autoplay && forceRemount && playFunction && this.phase !== "errored") {
+      if (this.renderOptions.autoplay && (initial ? this.story?.parameters?.autoplay ?? false : true) && forceRemount && playFunction && this.phase !== "errored") {
         window?.addEventListener?.("error", onError), window?.addEventListener?.("unhandledrejection", onUnhandledRejection), this.disableKeyListeners = !0;
         try {
           if (isMountDestructured ? await playFunction(context) : (context.mount = async () => {
