diff --git a/node_modules/@changesets/cli/dist/changesets-cli.cjs.js b/node_modules/@changesets/cli/dist/changesets-cli.cjs.js
index 53fc925..4cf6c1b 100644
--- a/node_modules/@changesets/cli/dist/changesets-cli.cjs.js
+++ b/node_modules/@changesets/cli/dist/changesets-cli.cjs.js
@@ -634,7 +634,13 @@ async function getPublishTool(cwd) {
   const pm = await packageManagerDetector.detect({
     cwd
   });
-  if (!pm || pm.name !== "pnpm") return {
+  if (!pm) return {
+    name: "npm"
+  };
+  if (pm.name === "yarn") return {
+    name: "yarn"
+  };
+  if (pm.name !== "pnpm") return {
     name: "npm"
   };
   try {
@@ -771,6 +777,9 @@ async function internalPublish(packageJson, opts, twoFactorState) {
   } = publishTool.name === "pnpm" ? await spawn__default["default"]("pnpm", ["publish", "--json", ...publishFlags], {
     env: Object.assign({}, process.env, envOverride),
     cwd: opts.cwd
+  }) : publishTool.name === "yarn" ? await spawn__default["default"]("yarn", ["npm", "publish", ...publishFlags], {
+    env: Object.assign({}, process.env, envOverride),
+    cwd: opts.cwd
   }) : await spawn__default["default"](publishTool.name, ["publish", opts.publishDir, "--json", ...publishFlags], {
     env: Object.assign({}, process.env, envOverride)
   });
diff --git a/node_modules/@changesets/cli/dist/changesets-cli.esm.js b/node_modules/@changesets/cli/dist/changesets-cli.esm.js
index 6c363c0..df8d412 100644
--- a/node_modules/@changesets/cli/dist/changesets-cli.esm.js
+++ b/node_modules/@changesets/cli/dist/changesets-cli.esm.js
@@ -596,7 +596,13 @@ async function getPublishTool(cwd) {
   const pm = await detect({
     cwd
   });
-  if (!pm || pm.name !== "pnpm") return {
+  if (!pm) return {
+    name: "npm"
+  };
+  if (pm.name === "yarn") return {
+    name: "yarn"
+  };
+  if (pm.name !== "pnpm") return {
     name: "npm"
   };
   try {
@@ -733,6 +739,9 @@ async function internalPublish(packageJson, opts, twoFactorState) {
   } = publishTool.name === "pnpm" ? await spawn$1("pnpm", ["publish", "--json", ...publishFlags], {
     env: Object.assign({}, process.env, envOverride),
     cwd: opts.cwd
+  }) : publishTool.name === "yarn" ? await spawn$1("yarn", ["npm", "publish", ...publishFlags], {
+    env: Object.assign({}, process.env, envOverride),
+    cwd: opts.cwd
   }) : await spawn$1(publishTool.name, ["publish", opts.publishDir, "--json", ...publishFlags], {
     env: Object.assign({}, process.env, envOverride)
   });
