diff --git a/node_modules/@popperjs/core/dist/esm/modifiers/computeStyles.js b/node_modules/@popperjs/core/dist/esm/modifiers/computeStyles.js
index 4434402..16bd6a9 100644
--- a/node_modules/@popperjs/core/dist/esm/modifiers/computeStyles.js
+++ b/node_modules/@popperjs/core/dist/esm/modifiers/computeStyles.js
@@ -22,8 +22,8 @@ function roundOffsetsByDPR(_ref) {
   var win = window;
   var dpr = win.devicePixelRatio || 1;
   return {
-    x: round(round(x * dpr) / dpr) || 0,
-    y: round(round(y * dpr) / dpr) || 0
+      x: round(x * dpr) / dpr || 0,
+      y: round(y * dpr) / dpr || 0,
   };
 }
 
@@ -88,6 +88,8 @@ export function mapToStyles(_ref2) {
     position: position
   }, adaptive && unsetSides);
 
+  ({x, y} = roundOffsets === true ? roundOffsetsByDPR({x, y}) : {x, y});
+
   if (gpuAcceleration) {
     var _Object$assign;
 
