diff --git a/node_modules/@web/storybook-builder/dist/index.js b/node_modules/@web/storybook-builder/dist/index.js
index 7d24e69..f0adef5 100644
--- a/node_modules/@web/storybook-builder/dist/index.js
+++ b/node_modules/@web/storybook-builder/dist/index.js
@@ -52,7 +52,7 @@ const bail = async () => {
     await (wdsServer === null || wdsServer === void 0 ? void 0 : wdsServer.stop());
 };
 exports.bail = bail;
-const start = async ({ startTime, options, router }) => {
+const start = async ({ startTime, options, router, server }) => {
     const previewDirOrigin = (0, path_1.join)((0, get_node_module_dir_1.getNodeModuleDir)('@storybook/preview'), 'dist');
     router.use('/sb-preview', express_1.default.static(previewDirOrigin, { immutable: true, maxAge: '5m' }));
     router.use(`/${rollup_plugin_prebundle_modules_1.PREBUNDLED_MODULES_DIR}`, express_1.default.static((0, path_1.resolve)(`./${rollup_plugin_prebundle_modules_1.PREBUNDLED_MODULES_DIR}`)));
@@ -90,7 +90,7 @@ const start = async ({ startTime, options, router }) => {
         wdsFinalConfig.open = `${protocol}://${host}:${port}${wdsFinalConfig.open}`;
     }
     // setup middleware mode
-    wdsFinalConfig.middlewareMode = true;
+    wdsFinalConfig.middlewareMode = { server };
     wdsFinalConfig.port = undefined;
     wdsFinalConfig.hostname = undefined;
     wdsServer = await (0, dev_server_1.startDevServer)({
