diff --git a/node_modules/lerna/dist/cli.js b/node_modules/lerna/dist/cli.js
index 8cd49ff..a036ba6 100755
--- a/node_modules/lerna/dist/cli.js
+++ b/node_modules/lerna/dist/cli.js
@@ -5760,7 +5760,7 @@ var require_git_add = __commonJS({
           if (!hasPrettier) {
             return;
           }
-          const prettierPath = import_path18.default.join(import_devkit2.workspaceRoot, "node_modules", "prettier");
+          const prettierPath = import_path18.default.join(import_devkit2.workspaceRoot, "node_modules", "@storybook", "cli", "node_modules", "prettier");
           resolvedPrettier = require(prettierPath);
         } catch {
           return;
