diff --git a/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncherPage.js b/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncherPage.js
index 25ee085..6c51474 100644
--- a/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncherPage.js
+++ b/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncherPage.js
@@ -25,10 +25,15 @@ class PlaywrightLauncherPage {
         await this.playwrightPage.setViewportSize({ height: 600, width: 800 });
         await this.playwrightPage.goto(url);
     }
+    stoppingSession = false;
     async stopSession() {
         const testCoverage = this.nativeInstrumentationEnabledOnPage
             ? await this.collectTestCoverage(this.config, this.testFiles)
             : undefined;
+        if (!this.stoppingSession) {
+            this.stoppingSession = true;
+            return { testCoverage };
+        }
         // navigate to an empty page to kill any running code on the page, stopping timers and
         // breaking a potential endless reload loop
         await this.playwrightPage.goto('about:blank');
